Documentation/admin/install-multi.md

```diff
@@ -0,0 +1,195 @@
+<br>
+<div class="alert alert-info" role="alert">
+<i class="fa fa-exclamation-triangle"></i><b> Note:</b> This documentation is for an alpha feature. For questions and feedback on the Multi-cluster Alpha program, email <a href="mailto:tectonic-alpha-feedback@coreos.com">tectonic-alpha-feedback@coreos.com</a>.
+</div>
+
+# Installing the multi-cluster registry
+
+Use Tectonic's multi-cluster registry to:
+* maintain a list of available clusters
+* enable cluster administrators to configure base security policies, and enforce other standardization policies across multiple clusters
+
+Use this guide to install and configure the registry.
+
+## Install the registry
+
+Install the multi-cluster registry by downloading and applying the example manifest:
```

Comment:
In my testing I found that the Pod takes longer than intuitive to starting `Running` if the Secret is created after these manifests are created. 

@zbwright WDYT of moving this step until after the `directory-sync` Secret is created?

Reply:
that makes more sense to me than doing it at the beginning. I'll move.